hack/lib/golang.sh

```diff
@@ -735,14 +741,14 @@ kube::golang::build_binaries() {
       for platform in "${platforms[@]}"; do (
           kube::golang::set_platform_envs "${platform}"
           kube::log::status "${platform}: build started"
-          kube::golang::build_binaries_for_platform ${platform}
+          kube::golang::build_binaries_for_platform "${platform}"
           kube::log::status "${platform}: build finished"
         ) &> "/tmp//${platform//\//_}.build" &
       done
 
       local fails=0
       for job in $(jobs -p); do
-        wait ${job} || let "fails+=1"
+        wait "${job}" || (( fails+=1 ))
```

Comment:
```
wait ${job} || let "fails+=1"
     ^----^ SC2086: Double quote to prevent globbing and word splitting.
               ^-----------^ SC2219: Instead of 'let expr', prefer (( expr )) .
```